DOCUMENTATION.md

```diff
@@ -2559,7 +2564,7 @@ The `logs` object yielded by this function emits the following events:
 
 - `line`: when a log line is received.
 - `clear`: when the logs are cleared.
-- `error`: when an error happens.
+- `error`: when an error has happened.
```

Comment:
This just came in when I rebased and rebuilt - I expect it's just because the documentation wasn't regenerated.